PlayerKit/Classes/RegularPlayer.swift

```diff
@@ -400,3 +408,57 @@ extension RegularPlayer: FillModeCapable
         }
     }
 }
+
+extension RegularPlayer: TextTrackCapable
+{
+    public func availableTextTracks() -> [TextTrackMetadata]
+    {
+        guard let group = self.player.currentItem?.asset.mediaSelectionGroup(forMediaCharacteristic: .legible) else
+        {
+            return []
```

Comment:
Is there a benefit here to pass back an empty array, instead of returning an optional?

Reply:
Yeah I went back and forth in my head about this. I came down on the side of an empty array being a better conceptual model since an optional array would give two distinct "no text tracks" states: `.none`, and `.some([])`.